Python/Product/Debugger.Concord/Proxies/Structs/PyCodeObject310.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
```

Comment:
Add copyright comments in the new files?

Reply:
Oh yeah, good idea. Didn't realize that.